llvm/include/llvm/ADT/GenericCycleInfo.h

```diff
@@ -261,6 +261,7 @@ template <typename ContextT> class GenericCycleInfo {
   const ContextT &getSSAContext() const { return Context; }
 
   CycleT *getCycle(const BlockT *Block) const;
+  CycleT *getCycleContaining(const BlockT *A, const BlockT *B) const;
```

Comment:
Please instead make this a member `CycleT *getSmallestCommonCycle(CycleT *A, CycleT *B) const` so that it is more generally useful.

Reply:
Done

Reply:
The idea was that the method would take Cycle arguments...